dns/pfSense-pkg-bind9/files/usr/local/pkg/bind.inc

```diff
@@ -513,7 +510,10 @@ EOD;
 							}
 						}
 						for ($y = 0; $y < sizeof($zone['row']); $y++) {
-							$hostname = (preg_match("/(MX|NS)/", $zone['row'][$y]['hosttype']) ? "@" : $zone['row'][$y]['hostname']);
+							$hostname = $zone['row'][$y]['hostname'];
+							if (preg_match("/(MX|NS)/", $zone['row'][$y]['hosttype']) && trim($hostname) == false) {
```

Comment:
According to trim docs it will never return false, only trimmed string.

http://php.net/manual/en/function.trim.php